admission_control/admission_control_proto/build.rs

```diff
@@ -1,19 +1,19 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-//! This compiles all the `.proto` files under `src/` directory.
-//!
-//! For example, if there is a file `src/a/b/c.proto`, it will generate `src/a/b/c.rs` and
-//! `src/a/b/c_grpc.rs`.
-
 fn main() {
-    let proto_root = "src";
-    let dependent_root = "../../types/src/proto";
-    let mempool_dependent_root = "../../mempool/src/proto/shared";
+    let protos = ["src/proto/admission_control.proto"];
+
+    let includes = [
+        "../../types/src/proto",
+        "src/proto",
+        "../../mempool/src/proto/shared",
+    ];
 
-    build_helpers::build_helpers::compile_proto(
```

Comment:
does this mean we'll be able to remove build_helpers from common/ btw?

Reply:
Yes i'll have a diff soon to finish cleaning up